slides/02-data-budget.qmd

```diff
@@ -24,59 +24,68 @@ knitr:
 
-## Data on tree frog hatching
+## Data on Chicago taxi trips
 
-![](images/Hatching-process.jpg)
+![](images/taxi.png)
 
-## Data on tree frog hatching
+::: footer
+Credit: <https://unsplash.com/photos/7_r85l4eht8>
+:::
+
+## Data on Chicago taxi trips
 
--   Red-eyed tree frog embryos can hatch earlier than their normal \~7 days if they detect potential predator threat!
--   Type `?stacks::tree_frogs` to learn more about this dataset, including references.
--   We are using a slightly modified version from stacks.
+-   The city of Chicago releases anonymized trip-level data on taxi trips in the city.
+-   We pulled a sample of 10,000 rides occurring in early 2022.
+-   Type `?modeldatatoo::data_taxi()` to learn more about this dataset, including references.
+-   We are using a slightly modified version from the modeldatatoo data.
 
 ```{r}
 library(tidymodels)
+library(modeldatatoo)
 
-data("tree_frogs", package = "stacks")
-tree_frogs <- tree_frogs %>%
-  mutate(t_o_d = factor(t_o_d),
-         age = age / 86400) %>%
-  filter(!is.na(latency)) %>%
-  select(-c(clutch, hatched))
+taxi <- data_taxi() %>%
+  mutate(month = factor(month, levels = c("Jan", "Feb", "Mar", "Apr"))) %>% 
+  select(-c(id, duration, fare, tolls, extras, total_cost, payment_type)) %>% 
+  drop_na()
 ```
 
-## Data on tree frog hatching
+## Data on Chicago taxi trips
 
 ::: columns
 ::: {.column width="60%"}
--   `N = 572`
--   A numeric outcome, `latency`
--   4 other variables
-    -   `treatment`, `reflex`, and `t_o_d` are **nominal** predictors
-    -   `age` is a **numeric** predictor
+-   `N = 10,000`
+-   A nominal outcome, `tip`, with levels `"yes"` and `"no"`
+-   6 other variables
+    -   `company`, `local`, and `dow`, and `month` are **nominal** predictors
+    -   `distance` and `hours` are **numeric** predictors
 :::
 
 ::: {.column width="40%"}
-![](images/Ac_2tads.jpg)
+![](images/taxi.png)
 :::
 :::
 
 :::notes
-- latency: How long it took the frog to hatch after being stimulated - i.e. after being poked by a blunt probe (in seconds).
+`tip`: Whether the rider left a tip. A factor with levels "yes" and "no".
+
+`distance`: The trip distance, in odometer miles.
+
+`company`: The taxi company, as a factor. Companies that occurred few times were binned as "other".
+
+`local`: Whether the trip started in the same community area as it began. See the source data for community area values.
 
-- treatment: Whether or not they got gentamicin, a compound that knocks out the embryo's lateral line (a sensory organ).
+`dow`: The day of the week in which the trip began, as a factor.
 
-- reflex: A measure of ear function (low, mid, full)
+`month`: The month in which the trip began, as a factor.
 
-- t_o_d: Time that the stimulus was applied (morning, afternoon, night)
+`hour`: The hour of the day in which the trip began, as a numeric.
 
-- age: Age at the time it was stimulated (in days)
 :::
 
-## Data on tree frog hatching
+## Data on chicago taxi trips
```

Comment:
```suggestion
## Data on Chicago taxi trips
```

Reply:
Thank you! [e35c41b](https://github.com/tidymodels/workshops/pull/108/commits/e35c41b98a0a91a1d4d3b54be04755f5962af808)